src/css/controls/imports/controlbar.less

```diff
@@ -96,9 +75,17 @@
     }
 }
 
-.jw-controlbar-left-group,
-.jw-controlbar-right-group {
-    white-space: nowrap;
+.jw-slider-horizontal {
+    background-color: transparent;
+}
+
+.jw-icon-cast {
```

Comment:
This collides with the declaration on [line 140](https://github.com/jwplayer/jwplayer/blob/cea51655b5f3a0c93a0fada53a3a6de66c46df29/src/css/controls/imports/controlbar.less#L140)